### PR TITLE
Added several TG items to the maint loots table.

### PR DESCRIPTION
**Pull request by SarmentiCampbell**
## About The Pull Request
Because apparently it wasn't added and it would be nice to get 'em. which by the way, maybe we should remove a few items from here, nah?

## How This Contributes To The Skyrat Roleplay Experience
maints modsuit

## Changelog

:cl:
add: Added several new items to Maints
/:cl:

### CODE REVIEW

**Comment by xXPawnStarrXx** [comment]
Please, Please add odius puffballs and Kronus vines. I'm saddened we can't get them yet. >:(

File: modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm [hunk 1]
```diff
@@ -65,5 +65,10 @@ GLOBAL_LIST_INIT(oddity_loot, list(//oddity: strange or crazy items
 	/obj/item/storage/toolbox/syndicate = 100,
 	/obj/item/storage/toolbox/infiltrator = 100,
 	/obj/item/toy/sprayoncan = 200,
-	/obj/item/warpwhistle = 5
+	/obj/item/warpwhistle = 5,
+	/obj/item/spear/grey_tide = 1,
```
Comment (RatFromTheJungle) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm:69]: No, this has absolutely no business being obtainable in round, grey_tide is one of the strongest items ingame.

File: modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm [hunk 2]
```diff
@@ -65,5 +65,10 @@ GLOBAL_LIST_INIT(oddity_loot, list(//oddity: strange or crazy items
...
+	/obj/item/shadowcloak = 1,
+	/obj/item/clothing/head/helmet/abductor = 1,
+	/obj/item/clothing/shoes/jackboots/fast = 1,
```
Comment (RatFromTheJungle) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm:72]: This neither, there's a reason we limit move-speed boosts, there shouldn't be 'makes you fast as fuck boots' just casually in maints.

File: modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm [hunk 3]
```diff
@@ -65,5 +65,10 @@ GLOBAL_LIST_INIT(oddity_loot, list(//oddity: strange or crazy items
...
+	/obj/item/clothing/suit/armor/reactive/table = 1
```
Comment (RatFromTheJungle) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm:73]: Also, no, this doesn't need to be findable in maints, assistants don't need an armor that tables themselves** if someone tries to touch them.

File: modular_skyrat/master_files/code/_globalvars/maint_loot_uncommon.dm [hunk 4]
```diff
@@ -10,6 +10,7 @@ GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
 		/obj/item/skillchip/glassblowing_master = 10,
 		/obj/item/skillchip/ceramic_master = 10,
 		/obj/item/skillchip/fishing_master = 10,
+		/obj/item/skillchip/brainwashing = 10,
```
Comment (RatFromTheJungle) [inline modular_skyrat/master_files/code/_globalvars/maint_loot_uncommon.dm:13]: Very much **NO**, brainwashing has 0, a fat 0 business being obtainable in round, even if it's 'rare', it's not a good fit here.

File: modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm (hunk 1 above)
Comment (Iamgoofball) [reply modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm:69]: this is fine if you guys didn't make maintenance loot way more available via trash piles

**Pull request closed without merging**